sdk/keyvault/azcertificates/client.go

```diff
@@ -70,7 +70,7 @@ type BeginCreateCertificateOptions struct {
 	Enabled *bool `json:"enabled,omitempty"`
 
 	// Application specific metadata in the form of key-value pairs
-	Tags map[string]string `json:"tags,omitempty"`
+	Tags map[string]*string `json:"tags,omitempty"`
```

Comment:
Is setting a value to 'nil' here equivalent to deleting it?

Reply:
Talked with @chlowell offline - we're kind of stuck just due to the fact that string can't be nil and (even if it's not valid today) we need to do it this particular way to handle any potential future change in the API.